.github/workflows/validate.yml

```diff
@@ -1,7 +1,6 @@
 name: Scheduled validator
 
-on:
-  push:
+on: [push, pull_request]
```

Comment:
If you're updating this, I would recommend updating the checkout action to use `v2` as well

Reply:
There are further minor updates that could be done in this file as well. But it's not relevant for this PR.

Reply:
Fixed with v2, and also the capitalisation of OPTIMADE